packages/protocol/contracts/stability/Exchange.sol

```diff
@@ -37,7 +38,20 @@ contract Exchange is IExchange, Initializable, Ownable, UsingRegistry {
     uint256 minimumReports
   );
 
+  event StableTokenSet(
+    address stable
+  );
+
+  event SpreadSet(
+    uint256 spread
```

Comment:
@m-chrzan `spread` and `reserveFraction` are of type `Fixidity.Fraction`, but this doesn't seem to be allowed in events. It resulted in a TypeError during compilation when I tried to use it here:

```
TypeError: This type is only supported in the new experimental ABI encoder. Use "pragma experimental ABIEncoderV2;" to enable the feature.
    FixidityLib.Fraction spread
```

Since the param given to `initialize` is a `uint256`, I went with that. Is that the right thing to do?

Reply:
Yep, that's exactly the way to do it! You can use `.unwrap()` on a Fixidity Fraction to get the internal representation.